src/celeritas/field/FieldDriver.hh

```diff
@@ -197,7 +196,7 @@ FieldDriver<StepperT>::find_next_chord(real_type       step,
             // Estimate a new trial chord with a relative scale
             step *= max(std::sqrt(options_.delta_chord / dchord), half());
         }
-    } while (!succeeded && (--remaining_steps > 0));
+    } while (!succeeded && --remaining_steps > 0);
```

Comment:
Is that an intentional removal of a redundant-but-might-be-useful-for-the-author-to-read parenthesis?  (i.e. I do not know why the original author had them there)

Reply:
Again consistency with the rest of the file